sdk/metric/sdk.go

```diff
@@ -301,32 +302,32 @@ func (s *syncInstrument) RecordOne(ctx context.Context, number api.Number, kvs [
 	h.RecordOne(ctx, number)
 }
 
-// New constructs a new SDK for the given batcher.  This SDK supports
-// only a single batcher.
+// New constructs a new Accumulator for the given integrator.  This Accumulator supports
+// only a single integrator.
 //
-// The SDK does not start any background process to collect itself
-// periodically, this responsbility lies with the batcher, typically,
+// The Accumulator does not start any background process to collect itself
+// periodically, this responsbility lies with the integrator, typically,
 // depending on the type of export.  For example, a pull-based
-// batcher will call Collect() when it receives a request to scrape
-// current metric values.  A push-based batcher should configure its
+// integrator will call Collect() when it receives a request to scrape
+// current metric values.  A push-based integrator should configure its
 // own periodic collection.
-func New(batcher export.Batcher, opts ...Option) *SDK {
+func New(integrator export.Integrator, opts ...Option) *Accumulator {
```

Comment:
Should this be renamed to `NewAccumulator`? It no longer is scoped by the package name (`sdk.New` doesn't return an `SDK` anymore).

Reply:
Done.